packages/visual-editor/src/utils/assemblyUtils.spec.ts

```diff
@@ -64,6 +67,11 @@ describe('deserializeAssemblyNode', () => {
       type: ASSEMBLY_NODE_TYPE,
       parentId: 'root',
       data: {
+        assembly: {
```

Comment:
[nitpick]
This test file is just a collection of dumb snapshot tests and not helpful at all to maintain our code or to understand what the util function is doing. You can tell that your new feature just require adding similar values all over the file without actually adding a separate test case that describes an tests any new behaviour that we added. Unit tests should test behaviour not snapshots. I hope we find the time soon to rewrite this entire file and make it right :(